### PR TITLE
Add RandomApply, RandomChoice, RandomOrder transformations

### PR DESCRIPTION
**Pull request by vfdev-5**
Some handful classes to randomly apply list of transformations:
1) Random apply: applies a list of transformation with a probability
```
RandomApply(
            [
                transforms.RandomRotation((-45, 45)),
                transforms.RandomHorizontalFlip(),
                transforms.RandomVerticalFlip(),
            ], proba=0.75
)
```
2) Random choice: choose randomly a single transformation from a list
3) Random order: apply transformations in a random order

### CODE REVIEW

**Comment by vfdev-5** [comment]
Any feedback on the PR is highly appreciated :)

File: torchvision/transforms/transforms.py [hunk 1]
```diff
@@ -261,6 +261,77 @@ def __repr__(self):
...
+        format_string += '\n)'
+        return format_string
+
+
+class RandomApply(RandomTransforms):
+    """Apply randomly a list of transformations with a probability
+
+    Args:
+        transforms (list or tuple): list of transformations
+        proba (float): probability
```
Comment (fmassa) [inline torchvision/transforms/transforms.py]: can you maybe make it `p`, so that it is consistent with other transforms?

Comment (vfdev-5) [reply torchvision/transforms/transforms.py]: Totally agree

**Comment by vfdev-5** [comment]
@fmassa fixed

**Pull request merged**